UI/js-src/lsmb/menus/Cache.js

```diff
@@ -0,0 +1,65 @@
+define(["dojo/_base/lang","dojo/when" /*=====, "dojo/_base/declare", "dojo/store/api/Store" =====*/],
+function(lang, when /*=====, declare, Store =====*/){
+
+// module:
+//		lsmb/menus/Cache
+
+var Cache = function(masterStore, cachingStore, options){

```

Comment:
Because the actual Dojo cache doesn't return a promise, forcing the user to manage the cache externaly. This one returns a promise, so sync or async are transparents.